app.js

```diff
@@ -30,7 +30,12 @@ const chosenScissorsElement = document.getElementById('chosen-scissors');
 
 const submitButton = document.getElementById('submit-button');
 
-// values
+// Computer values
+const drawnPaper = document.getElementById('computer-drawn-paper');
```

Comment:
it would be nice to make a bigger refactor and work on all variable names in this code. 
My proposal is (for training) - use objects. In example:
```
const playerElements = {
 paperButton: document.getElemen...,
  ...
}
```
And then you could have similar option for computer. Also - id `computer-paper` is for sure enough descriptive